packages/components/src/components/entities/actions.ts

```diff
@@ -234,15 +236,15 @@ export function getEntityTypeOptions(typeListSchemaQuery: SchemaQuery, typeSchem
  * @param allowParents are parents of this entity type allowed or not
  * @param filterArray (optional) set of filters to use for getting entity type options
  */
-export function getEntityTypeData(model: EntityIdCreationModel, schemaQueries: Map<string, SchemaQuery>, targetQueryName: string, allowParents: boolean, filterArray?: Array<Filter.IFilter>) : Promise<Partial<EntityIdCreationModel>> {
+export function getEntityTypeData(model: EntityIdCreationModel, schemaQueries: Map<string, EntityDataType>, targetQueryName: string, allowParents: boolean) : Promise<Partial<EntityIdCreationModel>> {
```

Comment:
Rename from `schemaQueries` to `dataTypes`?

Since `filterArray` has been removed, remove param documentation.

Reply:
Done and Done.